### ISSUE TITLE
logits_py incorrect in gumbel_softmax_vae_v2

### ISSUE THREAD

**Issue by backpropper** [ericjang/gumbel-softmax#9]
`logits_py` should be the log of the current `logits_py` or we can just give it as `probs` and not `logits` to the corresponding distribution.

**Comment by bobxwu** [comment]
Hi, do you know the meaning of the APIs **log_prob** and **prob**  in tf.distributions or tf.contrib.distributions?
They are quite confusing.
